build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
 		maven { url 'https://repo.spring.io/plugins-release' }
 	}
 	dependencies {
-		classpath 'io.spring.gradle:docbook-reference-plugin:0.3.1'
 		classpath 'org.asciidoctor:asciidoctor-gradle-plugin:1.5.8'
+		classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.15'
```

Comment:
It seems to me I've asked this question in other place, but let me to repeat it here anyway for better coverage and context.

Do we really still support PDF view form our Reference Manuals?
Is there a reason to keep this `alpha` dependency?

Reply:
We produce PDF for people who want to read it as a book and for people who want to read offline (on airplanes or whatever). I don't know why that's still alpha. It's been around for years now, so it seems like there should be a release version of it.